test/testcmdlineparser.cpp

```diff
@@ -1369,8 +1375,56 @@ class TestCmdlineParser : public TestFixture {
     void stdunknown2() {
         REDIRECT;
         const char *const argv[] = {"cppcheck", "--std=cplusplus11", "file.cpp"};
-        TODO_ASSERT_EQUALS(static_cast<int>(CmdLineParser::Result::Fail), static_cast<int>(CmdLineParser::Result::Success), static_cast<int>(parser->parseFromArgs(3, argv)));
-        TODO_ASSERT_EQUALS("cppcheck: error: unknown --std value 'cplusplus11'\n", "", logger->str());
+        ASSERT_EQUALS(static_cast<int>(CmdLineParser::Result::Fail), static_cast<int>(parser->parseFromArgs(3, argv)));
```

Comment:
Nit: It looks like we could use ASSERT_EQUALS_ENUM here now instead?

Reply:
Right, I suppose I made that change before I found the *_ENUM macro.

Reply:
There is just no TODO macro for enum so it still had the cast. Will be introduced shortly though.